### ISSUE TITLE
Browser is not working. Failed to execute default Web Browser

### ISSUE THREAD

**Issue by LuckyTechnology** [MrJackSpade/RG35XXP-XFCE#34]
I just tested the latest release and the Browser is not working. Failed to execute default Web Browser

**Comment by MrJackSpade** [comment]
I just reflashed everything and reinstalled from scratch, and its working for me.

Did you pull the code through GIT, or did you download it as a Zip?

I've seen the web browser fail to launch when the code was pulled through GIT and `autocrlf` was enabled, because GIT will bork the `.sh` files causing the configuration to fail during setup

**Comment by LuckyTechnology** [comment]
I used the zip. 

I tested it and it showed the error for me. 

I’ll reflash new and test it again. 

https://cxm.wetransfer.com/downloads/1307249a19990e4d8031aa3bf6d8955120241021024631/7c1d5e?t_exp=1761014792&t_lsid=4569bf87-832c-4fdf-8e44-22b12242ec43&t_network=link&t_rid=YXV0aDB8NjVmY2ZiYTdkYTA5OWY3NDMwMjNiZDY5&t_s=download_link&t_ts=1729478792

**Comment by MrJackSpade** [comment]
I don't have a RG35XXH to test but I will try flashing that image to my RG35XX+ and see what happens.

As far as I know (and until now) the images have worked perfectly fine across both devices/firmwares.

**Comment by LuckyTechnology** [comment]
Yeah since the changes made the past few days. 
The latest release from today seem to not work for me. Previous I had the zip with the install xfce and language was in Chinese lol 

Issue browser doesn’t open. Or may not exist for arm64

I try to install a browser using snapd but I’m not even sure if the store is working like Gnome :(

**Comment by MrJackSpade** [comment]
I was unable to find a browser that worked through any other method aside from bundling it with the application. Using things like `apt` install the wrong architecture.

The browser itself is in [this directory](https://github.com/MrJackSpade/RG35XXP-XFCE/tree/main/payload/usr/bin/firefox)

One of the browser files is zipped due to exceeding githubs file size limit.

The install script removes the existing Firefox directory, then copies the browser files from the payload into the firefox directory, and then unzips the zipped file.

Before you install XFCE you should be able to see all of those files in the payload directory, and after you run the installer, you should see them in /usr/bin/firefox

**Comment by LuckyTechnology** [comment]
lol I think I found the issue. Human error
I was following the instruction:
“Navigate to the Release/ARM-64 directory”

You didn’t push it into the release. It’s missing the file. 
/usr/bin/firefox

Only in the directory “The browser itself is in” [this directory](https://github.com/MrJackSpade/RG35XXP-XFCE/tree/main/payload/usr/bin/firefox)

**Comment by MrJackSpade** [comment]
Fuck me, you're right. Thats because I had a `.gitignore` entry to exclude `bin` directories, which excluded `/usr/bin` the Release directory.

I uploaded the original `payload` directory in the root before I made the installer and exe, and before I added the `bin` exclusion.

I've just pushed a commit that adds all of those files back into the release directory. Hopefully theres nothing else missing.
